qfieldsync/core/cloud_api.py

```diff
@@ -129,14 +129,18 @@ class CloudAuthMethod(Enum):
 
 
 def build_oauth2_auth_config(
-    auth_data: dict, related_uri: str, config_name: str = "qfieldcloud_sso"
+    auth_data: dict,
+    related_uri: str,
+    config_name: str = "qfieldcloud_sso",
+    persist: bool = True,
```

Comment:
Do we have control over this parameter name?  Can we prefix with SHOULD or other boolean indicator?